cmake/package_settings.cmake

```diff
@@ -57,13 +57,9 @@ include(CMakePackageConfigHelpers)
 configure_package_config_file(
   ${PROJECT_SOURCE_DIR}/cmake/openenclave-config.cmake.in
   ${CMAKE_BINARY_DIR}/cmake/openenclave-config.cmake
-  INSTALL_DESTINATION
```

Comment:
Not sure why my cmake-format wants to do that and that hasn't happened upstream.